Restore the previous LayerType value of View when using withLayer option

fabionuno opened the pull request:
When using withLayer option the view's layer type is set to
LAYER_TYPE_NONE at the end of animation independently of original value

fabionuno committed:
Restore the previous LayerType value of View when using withLayer option

When using withLayer option the view's layer type is set to
LAYER_TYPE_NONE at the end of animation independently of original value

blipinsk commented:
Hi, thanks for the PR!
Nice catch on the layer :+1:!
Could you please change the name of the `Listener's` field from `currentLayerType` to `mCurrenLayerType` to stay consistent with the naming across the library?

fabionuno committed:
Changes the currentLayerType variable name to mCurrentLayerType

changes the currentLayerType variable name to mCurrentLayerType to stay
consistent with the naming across the library

fabionuno commented:
Of course.
:thumbsup:

blipinsk commented:
Your PR has been included in Version 1.4.3 (should be visible in Maven Central in few hours). Once again thanks for the contribution! :+1:
